src/go/rdctl/pkg/factoryreset/empty.go

```diff
@@ -27,11 +27,3 @@ func CheckProcessWindows() (bool, error) {
 func KillRancherDesktop() error {
 	return fmt.Errorf("internal error: KillRancherDesktop shouldn't be called")
 }
-
```

Comment:
Should `pkg/factoryreset/empty.go` be renamed to `pkg/factoryreset/factory_reset_unix.go`?